runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java

```diff
@@ -120,7 +120,77 @@
+	ANTIDOTE_P(Type.POTION, "Antid+", "A+", ItemID.ANTIDOTE4, ItemID.ANTIDOTE3, ItemID.ANTIDOTE2, ItemID.ANTIDOTE1),
+	ANTIDOTE_PP(Type.POTION, "Antid++", "A++", ItemID.ANTIDOTE4_5952, ItemID.ANTIDOTE3_5954, ItemID.ANTIDOTE2_5956, ItemID.ANTIDOTE1_5958),
+	ANTIVENOM(Type.POTION, "Anti-V", "AV", ItemID.ANTIVENOM4, ItemID.ANTIVENOM3, ItemID.ANTIVENOM2, ItemID.ANTIVENOM1),
+	ANTIVENOM_P(Type.POTION, "Anti-V+", "AV+", ItemID.ANTIVENOM4_12913, ItemID.ANTIVENOM3_12915, ItemID.ANTIVENOM2_12917, ItemID.ANTIVENOM1_12919),
+
+	RELICYMS_BALM(Type.POTION, "Relicym", "R.B", ItemID.RELICYMS_BALM4, ItemID.RELICYMS_BALM3, ItemID.RELICYMS_BALM2, ItemID.RELICYMS_BALM1),
+	SANFEW_SERUM(Type.POTION, "Sanfew", "Sf", ItemID.SANFEW_SERUM4, ItemID.SANFEW_SERUM3, ItemID.SANFEW_SERUM2, ItemID.SANFEW_SERUM1),
+	ANTIFIRE(Type.POTION, "Antif", "Af", ItemID.ANTIFIRE_POTION4, ItemID.ANTIFIRE_POTION3, ItemID.ANTIFIRE_POTION2, ItemID.ANTIFIRE_POTION1),
+	EXTENDED_ANTIFIRE(Type.POTION, "E.Antif", "E.Af", ItemID.EXTENDED_ANTIFIRE4, ItemID.EXTENDED_ANTIFIRE3, ItemID.EXTENDED_ANTIFIRE2, ItemID.EXTENDED_ANTIFIRE1),
+	SUPER_ANTIFIRE(Type.POTION, "S.Antif", "S.Af", ItemID.SUPER_ANTIFIRE_POTION4, ItemID.SUPER_ANTIFIRE_POTION3, ItemID.SUPER_ANTIFIRE_POTION2, ItemID.SUPER_ANTIFIRE_POTION1),
+	EXTENDED_SUPER_ANTIFIRE(Type.POTION, "ES.Antif", "ES.Af", ItemID.EXTENDED_SUPER_ANTIFIRE4, ItemID.EXTENDED_SUPER_ANTIFIRE3, ItemID.EXTENDED_SUPER_ANTIFIRE2, ItemID.EXTENDED_SUPER_ANTIFIRE1),
+
+	SERUM_207(Type.POTION, "Ser207", "S7", ItemID.SERUM_207_4, ItemID.SERUM_207_3, ItemID.SERUM_207_2, ItemID.SERUM_207_1),
+	SERUM_208(Type.POTION, "Ser208", "S8", ItemID.SERUM_208_4, ItemID.SERUM_208_3, ItemID.SERUM_208_2, ItemID.SERUM_208_1),
+	COMPOST(Type.POTION, "Compost", "Cp", ItemID.COMPOST_POTION4, ItemID.COMPOST_POTION3, ItemID.COMPOST_POTION2, ItemID.COMPOST_POTION1),
+
+	// Unfinished Potions
+	GUAM_POTION(Type.POTION, "Guam", "G", ItemID.GUAM_POTION_UNF),
+	MARRENTILL_POTION(Type.SEED, "Marren", "M", ItemID.MARRENTILL_POTION_UNF),
+	TARROMIN_POTION(Type.SEED, "Tarro", "TAR", ItemID.TARROMIN_POTION_UNF),
+	HARRALANDER_POTION(Type.SEED, "Harra", "H", ItemID.HARRALANDER_POTION_UNF),
+	RANARR_POTION(Type.SEED, "Ranarr", "R", ItemID.RANARR_POTION_UNF),
+	TOADFLAX_POTION(Type.SEED, "Toad", "TOA", ItemID.TOADFLAX_POTION_UNF),
+	IRIT_POTION(Type.SEED, "Irit", "I", ItemID.IRIT_POTION_UNF),
+	AVANTOE_POTION(Type.SEED, "Avan", "A", ItemID.AVANTOE_POTION_UNF),
+	KWUARM_POTION(Type.SEED, "Kwuarm", "K", ItemID.KWUARM_POTION_UNF),
+	SNAPDRAGON_POTION(Type.SEED, "Snap", "S", ItemID.SNAPDRAGON_POTION_UNF),
+	CADANTINE_POTION(Type.SEED, "Cadan", "C", ItemID.CADANTINE_POTION_UNF),
+	LANTADYME_POTION(Type.SEED, "Lanta", "L", ItemID.LANTADYME_POTION_UNF),
+	DWARF_WEED_POTION(Type.SEED, "Dwarf", "D", ItemID.DWARF_WEED_POTION_UNF),
+	TORSTOL_POTION(Type.SEED, "Torstol", "TOR", ItemID.TORSTOL_POTION_UNF);
```

Comment:
Unless I'm misunderstanding, these should be `Type.POTION` instead of `Type.SEED`.

Reply:
Yep, that’s what I get for copying and pasting the seed table. I can fix it tomorrow